packages/wxt/typedoc.json

```diff
@@ -15,7 +15,6 @@
     "src/utils/define-wxt-plugin.ts",
     "src/utils/match-patterns.ts",
     "src/utils/storage.ts",
-    "src/browser.ts",
```

Comment:
Including this in the API docs was causing the entire `@types/chrome` JSDoc to be included, which was taking a long time an inflating the knowledge files. So I've removed this page from the API reference for now.